src/common/copilot/IntelligenceApiService.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import fetch, { RequestInit } from "node-fetch";
+import {sessionID} from "./PowerPagesCopilot";
+import { ActiveFileParams, InvalidResponse, NetworkError } from "./constants";
+
+
+export async function sendApiRequest(userPrompt: string, activeFileParams: ActiveFileParams, orgID:string, apiToken:string) {
+
+    const region = 'test';
+    let aibEndpoint = '';
+
+    //TODO: Add prod endpoint
+    switch (region) {
+        case 'test':
+         aibEndpoint = `https://aibuildertextapiservice.wus-il001.gateway.test.island.powerapps.com/v1.0/${orgID}/appintelligence/chat`
+          break;
+        default:
+          break;
+      }
+ 
```

Comment:
Can you modularize a bit more. 
1) getIntelligenceApiEndpoint and 
2) requestIntelligenceApi